### ISSUE TITLE
What about MNIST dataset?

### ISSUE THREAD

**Issue by yuletian** [carpedm20/BEGAN-tensorflow#12]
The model can not show any convergence clues in MNIST dataset during my training.
![agg_d_mnist](https://cloud.githubusercontent.com/assets/11909842/25035791/445aa7b0-2122-11e7-92f0-8a5da808969f.png)
after 1600 batch of training. It just remain no change with serious mode collapse.
something wrong?

**Comment by carpedm20** [comment]
Did you used the most recent code after b6dfb250063f86276ca5112cfc683777558202d4? I recently changed lots of code.

**Comment by yuletian** [comment]
I have updated the code already. Those images were generated with newest code. To be notice that, the k remain 0 during training. Is it make sense let k below 0?

**Comment by carpedm20** [comment]
No it should increase at first. `gamma` and `learning rate` is quite sensitive because they do affect the update of `k`. Because the dataset is different, the loss scale will be different from CelebA at the initial learning phase so you may need to change those hyperparameters.

**Comment by yuletian** [comment]
OK, I will try different learning rate and show the result here later.
